app/models/event.rb

```diff
@@ -4,4 +4,7 @@ class Event < ApplicationRecord
   validates :place_title, length: { maximum: 30 }
   validates :description, length: { minimum: 30 }
   validates_with StartEndTimeValidator
+
+  has_many :items, dependent: :delete_all
```

Comment:
Are you sure you want to use `delete_all` option instead of `destroy`?
Do you know the difference?

Reply:
delete_all -- related objects will be deleted directly from the database (without the call: destroy)
destroy -- will be called on related objects.

Reply:
+ / - not understandable

Reply:
@ZhenyaZhak let's use `destroy` here to delete all associated objects.
In our case, there'll be no difference (we don't have any callbacks yet and we won't have large datasets).